torch/csrc/profiler/util.cpp

```diff
@@ -97,6 +97,36 @@ bool softAssertRaises() {
   return soft_assert_raises_.value_or(false);
 }
 
+void logSoftAssert(
+    const char* func,
+    const char* file,
+    uint32_t line,
+    const char* cond,
+    const char* args) {
+#ifdef USE_KINETO
+  std::stringstream error;
+  error << cond << " SOFT ASSERT FAILED at " << file
```

Comment:
nit: for future reference this is a good place to use `fmt`. (But not worth changing since you have CI mostly done.)

Reply:
Great point! changing it to use fmt!